input/pages/qdm-to-qicore.md

```diff
@@ -798,16 +806,16 @@ _present on admission_ flag for each diagnosis. Therefore, the [Claim.diagnosis.
 [Claim.diagnosis.diagnosis\[x\]](StructureDefinition-qicore-claim-definitions.html#Claim.diagnosis.diagnosis[x]) defines which diagnoses are _present on admission_. The [Claim.procedure.sequence](StructureDefinition-qicore-claim-definitions.html#Claim.procedure.sequence) = 1
 plus [Claim.procedure.procedure\[x\]](StructureDefinition-qicore-claim-definitions.html#Claim.procedure.procedure[x]) defines a _principal procedure_.
 
-For this reason, QI-Core STU6 no longer includes Encounter.diagnosis in the Key Element Table of
-the profile. This QI-Core version aligns with the US Core 6.1.0 using [Encounter.reasonCode](StructureDefinition-qicore-encounter-definitions.html#Encounter.reasonCode) and
+For this reason, QI-Core STU7 no longer includes Encounter.diagnosis in the Key Element Table of
```

Comment:
For this reason, QI-Core STU7 **does not** includes Encounter.diagnosis in the Key Element